pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[build-system]
+requires = [
+    "setuptools",
+    "wheel",
+    "protoc-wheel-0==24.4",
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "open-simulation-interface"
+description = "ASAM Open Simulation Interface Python Bindings."
+authors = [
+    {name = "ASAM Open Simulation Interface Project", email = "osi@asam.net"},
+]
+maintainers = [
+    {name = "ASAM Open Simulation Interface Project", email = "osi@asam.net"},
+]
+dependencies = [
+    "protobuf>=4.24.4",
```

Comment:
better == ?

Reply:
Conservatively `==` would likely be better, however there might be some leeway here, which is why I have for now left it at `>=`, but might change to `==` prior to finalizing the PR.